.pipelines/e2e-windows.yaml

```diff
@@ -41,6 +41,8 @@ stages:
       windowsImage: 2019-containerd
       windowsOSSKU: Windows2019
       windowsVMSize: Standard_D2s_v3
+      windowsDistro: aks-windows-2019-containerd
```

Comment:
Can you help to add a README for how to generate and update the test images, how to generate the template, the brief introduction of windows e2e, etc.?

Reply:
Could I create a new task rather than in this pr to do it? I think it may take some time.